src/Knp/Menu/Renderer/ListRenderer.php

```diff
@@ -242,9 +237,8 @@ protected function format(string $html, string $type, int $level, array $options
 
             case 'li':
                 $spacing = $level * 4 - 2;
-                break;
         }
 
-        return \str_repeat(' ', $spacing).$html."\n";
+        return \str_repeat(' ', $spacing).$html.\PHP_EOL;
```

Comment:
why changing the line ending ?

Reply:
it's just the same, only using more explicit constant

Reply:
no, on windows, it is not the same

Reply:
I think that can be safe using Windows end of line character if you're on Windows.

Reply:
`ListRendererTest` will fail on Windows with this change. 🙃 

I wouldn't expect this library to produce different output on Windows than on Unix, tbh. I think this change should be reverted.